build.rs

```diff
@@ -154,7 +170,26 @@ fn build(sdk_path: Option<&str>, target: &str) {
         .map(|h| format!("#include <{}>\n", h))
         .collect();
 
-    builder = builder.header_contents("coreaudio.h", &meta_header.concat());
+    let fixes = if target.contains("apple-visionos") {
+        vec![
+            // /Applications/Xcode.app/Contents/Developer/Platforms/XROS.platform/Developer/SDKs/XROS1.1.sdk/System/Library/Frameworks/AudioToolbox.framework/Headers/AudioToolbox.h:43:11: fatal error: 'AudioToolbox/AudioFileComponent.h' file not found
+            "#define TARGET_OS_IPHONE true\n",
+            // https://github.com/phracker/MacOSX-SDKs/blob/master/MacOSX10.13.sdk/usr/include/MacTypes.h#L289
+            // /Applications/Xcode.app/Contents/Developer/Platforms/XROS.platform/Developer/SDKs/XROS1.1.sdk/System/Library/Frameworks/CoreMIDI.framework/Headers/MIDIServices.h:1633:8: error: unknown type name 'ItemCount'
+            "typedef unsigned long ItemCount;\n",
+            "typedef unsigned long ByteCount;\n",
+        ]
+    } else {
+        vec![""]
+    };
+
+    let contents = format!("{}{}", fixes.concat(), meta_header.concat());
+
+    println!("=============================");
+    println!("{}", &contents);
+    println!("=============================");
+
+    builder = builder.header_contents("coreaudio.h", &contents);
```

Comment:
Hmm. I'm confused by the need for this.

Reply:
`#define TARGET_OS_IPHONE true` hack for AudioUnit to pass on CI.

`ItemCount` and `ByteCount` are out of scope. I'll see if `bindgen` can be used to make it look more standard.

Looking into the `AVFAudio` and `AVAudioSession` so might make few more changes here before it's ready for the clean up.

Reply:
It's unclear why I didn't need these over in #103.

> Looking into the AVFAudio and AVAudioSession so might make few more changes here before it's ready for the clean up.

Sounds good. I feel like this may be why I was running into a crash in https://github.com/RustAudio/coreaudio-rs/pull/117.